qubesadmin/tools/qvm_ls.py

```diff
@@ -330,7 +330,10 @@ def calc_used(vm, volume_name):
     return '{}%'.format(usage * 100 // size)
 
 
-# todo maxmem
+# Show hyphen if maxmmem is zero (for HVMs)
+Column('MAXMEM',
+    attr=(lambda vm: vm.maxmem if vm.virt_mode != "hvm" else '-'),
```

Comment:
> 1. Why use a symbol `-` rather than `0` itself?

I also considered 'N/A'. I personally  hyphen or N/A should imply as not-applicable as maxmem does not make sense for HVMs. There are similar cases for the other fields.

Reply:
maxmem very much does make sense for HVM. It doesn't make sense for qubes with dynamic memory management disabled, which happen to be the case for the only HVMs in the default configuration.

Reply:
I think it is better to remove maxmem custom implementation from this BR since it is not relevant to the original target of this PR.